iocore/net/UnixNet.cc

```diff
@@ -682,8 +682,9 @@ NetHandler::_close_vc(UnixNetVConnection *vc, ink_hrtime now, int &handle_event,
     // create a dummy event
     Event event;
     event.ethread = this_ethread();
-    vc->handleEvent(EVENT_IMMEDIATE, &event);
-    ++handle_event;
+    if (vc->handleEvent(EVENT_IMMEDIATE, &event) == EVENT_DONE) {
+      ++handle_event;
+    }
   }
 }
 
```

Comment:
I guess this makes a bit more sense.  As far as I can tell the handle_event variable is only used for a Debug statement, so the change isn't crucial one way or another.